param/optical_flow_params_1.1.yaml

```diff
@@ -0,0 +1,54 @@
+# Settings for the optical flow estimator
+optical_flow_estimator:
+    # fov of the camera
+    fov: 1.07
+
+    # Minimum altitude to run the detector
+    min_estimation_altitude: 0.3
+
+    # Window size for optical flow
+    win_size: 20
+
+    # Levels of the pyramid to run the optical flow at
+    max_level: 3
+
+    # Number of iterations to run the optical flow
+    iters: 20
+
+    # Number of points to track
+    points: 100
+
+    # Quality level for the corner detector
+    quality_level: 0.01
+
+    # Min distance between points for the corner detector
+    min_dist: 20
+
+    # Image scale factor
+    scale_factor: 0.5
+
+    # Debug settings for the optical flow estimator
+    # Publish image with the velocity vectors drawn on it
+    debug_vectors_image: true
```

Comment:
Debug settings here need to be updated with new stuff from optical flow

Reply:
Fixed